bin/main.ml

```diff
@@ -38,6 +38,7 @@ let all : _ Cmdliner.Cmd.t list =
     ; Init.group
     ; Promotion.group
     ; Pkg.group
+    ; Pkg.Alias.group
```

Comment:
I personally would rather create a `Package` module and define the commands there (exposing `subcommands` and `info`) but I can see the point in not cluttering the public interface.

Reply:
This idea is to not duplicate the code and one of them is going to disappear at some point ^^